README.md

```diff
@@ -101,19 +107,25 @@ kallsyms and find the kernel image of the running kernel from several
 potential directories; for instance, `"/boot/"` and `"/usr/lib/debug/boot/"`.
 
 ```rust,ignore,compile_fail
-	let sym_srcs = [SymbolSrcCfg::Kernel { kallsyms: None, kernel_image: None }];
+	let sym_srcs = [
+    SymbolSrcCfg::Kernel(cfg::Kernel { kallsyms: None, kernel_image: None })
+  ];
 ```
 
 ### A list of ELF files
 
 You can still provide a list of ELF files and their base addresses if necessary.
 
 ```rust,ignore,compile_fail
-	let sym_srcs = [SymbolSrcCfg::Elf { file_name: PathBuf::from("/lib/libc.so.xxx"),
-	                                    base_address: 0x1f005d },
-	                SymbolSrcCfg::Elf { fie_name: PathBuf::from("/path/to/my/binary"),
-	                                    base_address: 0x77777 },
-	                ......
+	let sym_srcs = [
+    SymbolSrcCfg::Elf(cfg::Elf {
```

Comment:
nit: same here, indentation looks off

Reply:
I convert the README to use spaces instead of tabs.